### DISCUSSION TITLE
Welcome to docable-notebooks Discussions!

### DISCUSSION THREAD

**Discussion by ssmirr** [ottomatica/docable-notebooks#29] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about. ❔ 
  * Share ideas. 💡 
  * Engage with other community members. 💬 
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sharpninja** [comment]
Are there plans to integrate with the Dotnet Interactive?

**Reply by ssmirr** [reply]
We haven't considered integrating with Dotnet Interactive, _but_ we should be able to add support for running C# scripts in the `script` cells which can be run with a play button similar to Dotnet Interactive.

Can you please share more about your usecase? Are there other Dotnet Interactive features that you need other than running C# scripts?

btw thanks for posting on our discussions @sharpninja!

---

if you're not familiar with our `script` cells, here is a demo for javascript in a `script` cell:

<p align="center">
  <img src="https://github.com/ottomatica/docable-notebooks/raw/master/docs/img/docable-edit-script.gif"  width="70%">
</p>
